docs/source/design.rst

```diff
@@ -1,72 +1,72 @@
 rclUE Design
 ============
 
-Unreal Architecture 
+Unreal Architecture
 --------------------
-One of the core features of Unreal Engine is the ability to 
-create new objects and expose properties and functions to 
-the editor by way of panel integration (i.e. for changing data properties) 
+One of the core features of Unreal Engine is the ability to
+create new objects and expose properties and functions to
+the editor by way of panel integration (i.e. for changing data properties)
 or Blueprint, for scripting logic.
 
 Unreal’s type system favors inheritance over composition,
-and requires that (ultimately) all classes that work with 
+and requires that (ultimately) all classes that work with
 these features derive from a base UObject class. These UObjects are managed memory,
```

Comment:
... are managed by memory ...

Reply:
Thanks, @Antho1426RR, would be also convenient for the PR author if the comment is given as source suggestion I guess

Reply:
Sure, it would be more convenient indeed to make suggestions rather than comments in this case (as suggested at "https://haacked.com/archive/2019/06/03/suggested-changes/" I presume).